sync/_databricks.py

```diff
@@ -240,6 +307,17 @@ def _get_cluster_report(
     raise NotImplementedError()
 
 
+def _create_cluster_report(
+        cluster: dict,
+        cluster_info: dict,
+        cluster_activity_events: dict,
+        tasks: List[dict],
```

Comment:
I think these type hints need to be `Dict` to maintain the older python compatibility.

Reply:
Interesting. Theres a nice mix between the two already in the library. ill go with Dict though

Reply:
Oh hm, maybe it's only an issue when you try to type the contents of the dict. E.g. I'm almost certain that `dict[str]` would fail ... but maybe just a play `dict` is fine.